Update README.md

dfk7677 opened the pull request:
Just a few corrections.

rigor789 commented on README.md:
```diff
@@ -168,7 +168,7 @@ The duration (in milliseconds) of the scrolling animation.
 
 #### easing 
-The easing to be used when animating. Read more in the [Easing section](#easing1). 
+The easing to be used when animating. Read more in the [Easing section](#easing-1). 
```
This is a tricky one, because github uses`easing-1` but docute (the docs website) generates it as `easing1`. My idea is to just set the `## Easing` part to `<h2 id="easing-detailed">Easing</h2>` and use that as the id to be consistent on both github and the docs site. Can you please add these changes, so we can merge it all at once?

dfk7677 commented:
Thanks for your comment. I replaced the markdown with html in the _easing heading_. I also replaced the markdown for the _options link_ with html, which for some reason did not convert to a link when parsed.

rigor789 commented:
Thanks!
